module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -263,7 +262,7 @@ public function getAvailability($overDriveId)
             $availabilityUrl .= "$overDriveId/availability";
             $res = $this->callUrl($availabilityUrl);
 
-            if ($res->errorCode == 'NotFound') {
+            if (isset($res->errorCode) and $res->errorCode == 'NotFound') {
```

Comment:
You can simplify some conditions in this class by using null coalescing instead of isset checks. For example, this could become:

`if (($res->errorCode ?? '') == 'NotFound') {`